router/server.go

```diff
@@ -33,11 +33,11 @@ func NewRouter(ctx context.Context, h models.HandlerInterface, port int) *Router
 	mux.Handle("/api/k8sconfig/ping", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.KubernetesPingHandler))))
 	mux.Handle("/api/mesh/scan", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.InstalledMeshesHandler))))
 
-	mux.Handle("/api/load-test", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.LoadTestHandler))))
-	mux.Handle("/api/load-test-smps", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.LoadTestUsingSMPSHandler))))
-	mux.Handle("/api/load-test-prefs", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.LoadTestPrefencesHandler))))
-	mux.Handle("/api/results", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.FetchResultsHandler))))
-	mux.Handle("/api/result", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.GetResultHandler))))
+	mux.Handle("/api/perf/load-test", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.LoadTestHandler))))
```

Comment:
This just changes the routes on the server while as on UI the prefix perf is not added. @leecalcote Do we just need a change in routes at server level or at the user level too? As currently, there is no change in routes for a user who is accessing Meshery through UI.

Reply:
@kushthedude of the UI code affected here, it all seems to haven updated. From the tests that I did, the this updates worked.

Reply:
@kushthedude I understand what you mean now - not the API endpoints, but the UI routes. Got it. Thank for you diligently accounting for these as well. Let's give further thought to those as new UI pages come forth.

Reply:
Cool!